htdocs/includes/db_lib.php

```diff
@@ -2826,7 +2826,7 @@ public function addToDb()
 		# Updates an existing measure entry in DB
 		$saved_db = DbUtil::switchToLabConfigRevamp();
 		$query_string = 
-			"INSERT INTO measure (name, range, unit) ".

```

Comment:
`name` and `range` are reserved words in MySQL, so when you query columns with those names you need to escape them with backticks.